src/common/index.js

```diff
@@ -42,4 +42,5 @@ export { default as ViewPlaceholder } from './ViewPlaceholder';
 export { default as WebLink } from './WebLink';
 export { default as ZulipButton } from './ZulipButton';
 export { default as ZulipStatusBar } from './ZulipStatusBar';
+export { default as SelectionList } from './SelectionList';
 export { default as ZulipSwitch } from './ZulipSwitch';
```

Comment:
Notice these are alphabetized :slightly_smiling_face: -- please keep them that way